meinberlin/apps/budgeting/phases.py

```diff
@@ -55,6 +55,21 @@ class RatingPhase(phases.PhaseContent):
     }
 
 
+class VotingPhase(phases.PhaseContent):
+    app = apps.Config.label
+    phase = 'voting'
+    view = views.ProposalListView
+
+    name = _('How do you like the submitted proposals?')
+    description = _('Vote for up to 5 proposals for the participatory '
+                    'budget.')
+    module_name = _('participatory budgeting 3 phases')
+
+    features = {
+        'vote': (models.Proposal,)
+    }
+
+
 phases.content.register(RequestPhase())
 phases.content.register(CollectPhase())
 phases.content.register(RatingPhase())
```

Comment:
@Rineee I checked it out and it looks wonderful in the dashboard. So I'm wondering, why all the other phases are registered here and the VotingPhase isn't. What shouldn't be working then?

Reply:
Good catch. Saving the phases in the dashboard didn't work without it.